prow/plugins/blunderbuss/blunderbuss_test.go

```diff
@@ -197,14 +198,15 @@ var (
 		"e.go":  "5",
 		"ee.go": "5",
 	}
-	reviewers = map[string]sets.String{
-		"a.go": sets.NewString("al"),
-		"b.go": sets.NewString("al"),
-		"c.go": sets.NewString("charles"),
+	reviewers = map[string]layeredsets.String{
+		"a.go": layeredsets.NewString("al"),
+		"b.go": layeredsets.NewString("al"),
+		// "c.go": layeredsets.NewString("charles", "ben"), // fails with this one
```

Comment:
@alvaroaleman here

Reply:
I don't understand it, this is a commented testcase, not a testcase?

Reply:
Ok, gimme some time, as including it in a permanent test case requires some refactoring

Reply:
Its completely fine to add a new testcase for this that works different from the existing testcase (and probably even better, as refactoring the tests introduces some risk of regressions).

Reply:
> Its completely fine to add a new testcase for this that works different from the existing testcase (and probably even better, as refactoring the tests introduces some risk of regressions).

done!